mock/mock.go

```diff
@@ -332,6 +332,8 @@ func (m *Mock) TestData() objx.Map {
 */
 
 // Test sets the test struct variable of the mock object
+// Test should not be called on an object that is going to be used in a
```

Comment:
Need to add a `.` on the previous sentence:
![image](https://github.com/user-attachments/assets/48b99283-5cef-4084-99e3-4f3cda11b61f)

Reply:
While here, we might improve the existing doc?
``go
// Test sets the [TestingT] on which errors will be reported, otherwise errors will cause a panic.